test/index.js

```diff
@@ -95,6 +95,34 @@ describe('validations', () => {
     falsy.forEach((item) => assert.isFalse(validations.isPhone(item), `Failed for ${item}`));
   });
 
+  it('isUrl', () => {
+    const truthy = [
+      'http://nebenan.de',
+      'https://nebenan.de',
+      'https://www.nebenan.de',
+      'https://www.nebenan.com',
+      'nebenan.de',
+      'www.nebenan.de',
+      'nebenan.de/about',
+      'org.nebenan.de',
+      'org.nebenan-1.de',
+      'nebenan.de/more/than/one/path',
+    ];
+
+    const falsy = [
+      'http://nebenan.d',
+      'http://nebenande',
+      'http://nebenan. de',
+      'https:// nebenan.de',
+      'www.neben@an.de',
+      'https//nebenan.de',
+      'https:/nebenan.de',
+    ];
+
+    truthy.forEach((item) => assert.isTrue(validations.isUrl(item), `Passed for ${item}`));
```

Comment:
don't pack too much logic into tests. rather write the assertions by hand

`assert.isTrue(valications.isUrl('http://nebenan.de'))`

Reply:
you should always try to have as little logic as possible in your tests

otherwise you need tests to test your tests, just to be sure the logic inside the tests works 😀